uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/manage-leave-requests.component.js

```diff
@@ -254,36 +254,43 @@ define([
     }
 
     /**
-     * Loads the managees and calls loadLeaveRequests()
+     * Loads the leave requests for the given data type
      *
+     * @param {String} dataType - which data to load, table or filter or both
      * @return {Promise}
      */
-    function loadManageesAndLeaves () {
-      vm.loading.content = true;
+    function loadData (dataType) {
```

Comment:
why are you calling this function `loadData` if all it's doing is to load leave requests? Just call it something like `loadLeaveRequestsForSection` or something similar (the name is just a draft)